src/js/components/AppListComponent.jsx

```diff
@@ -123,6 +126,10 @@ var AppListComponent = React.createClass({
       "hidden": state.fetchState !== States.STATE_ERROR
     });
 
+    var unauthorizedClassSet = classNames({
```

Comment:
Generally good point, but noAppClassSet hasn't the 'text-danger'-class.
And see other comment above.
Don't let us change this strange behavior in this PR.